packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/components/Stages/Stage/Stage.js

```diff
@@ -40,44 +24,52 @@ function Stage({ id, name, index, canDelete, isOpen: isOpenDefault = false }) {
   const dispatch = useDispatch();
 
   return (
-    <StyledAccordion>
-      <Accordion size="S" variant="primary" onToggle={() => setIsOpen(!isOpen)} expanded={isOpen}>
-        <AccordionToggle
-          title={name}
-          togglePosition="left"
-          action={
-            canDelete ? (
-              <DeleteButton
-                noBorder
-                onClick={() => dispatch(deleteStage(id))}
-                label={formatMessage({
-                  id: 'Settings.review-workflows.stage.delete',
-                  defaultMessage: 'Delete stage',
-                })}
-                icon={<Trash />}
-              />
-            ) : null
-          }
-        />
-        <AccordionContent padding={6} background="neutral0">
-          <Grid gap={4}>
-            <GridItem col={6}>
-              <TextInput
-                {...field}
-                id={fieldIdentifier}
-                value={name}
-                label={formatMessage({
-                  id: 'Settings.review-workflows.stage.name.label',
-                  defaultMessage: 'Stage name',
-                })}
-                error={meta.error ?? false}
-                onBlur={(event) => dispatch(updateStage(id, { name: event.target.value }))}
-              />
-            </GridItem>
-          </Grid>
-        </AccordionContent>
-      </Accordion>
-    </StyledAccordion>
+    <Accordion
+      size="S"
+      variant="primary"
+      onToggle={() => setIsOpen(!isOpen)}
+      expanded={isOpen}
+      shadow="tableShadow"
+    >
+      <AccordionToggle
+        title={name}
+        togglePosition="left"
+        action={
+          canDelete ? (
+            <IconButton
+              backgroundColor="transparent"
+              noBorder
+              onClick={() => dispatch(deleteStage(id))}
+              label={formatMessage({
+                id: 'Settings.review-workflows.stage.delete',
+                defaultMessage: 'Delete stage',
+              })}
+              icon={<Trash />}
+            />
+          ) : null
+        }
+      />
+      <AccordionContent padding={6} background="neutral0">
+        <Grid gap={4}>
+          <GridItem col={6}>
+            <TextInput
```

Comment:
If you want it to work on `blur` or typical `enter` stuff, why not just wrap it in a `form`? then you can do `e.preventDefault()` and do the rest...? Or can you not do that because you're using Formik?

Reply:
suppose thats why people don't like form submits being disabled 😛

Reply:
Yep. Wonder if we should track / discuss that at some point with design?